tests/app/dao/test_ft_billing_dao.py

```diff
@@ -44,12 +44,12 @@
 
 
 # This take a long time to execute.  Can it be replaced with a bulk insert?
-def set_up_yearly_data(sample_service, sample_template, sample_ft_billing):
+def set_up_yearly_data(sample_service, sample_template, sample_ft_billing, years):
     service = sample_service()
     sms_template = sample_template(service=service, template_type=SMS_TYPE)
     email_template = sample_template(service=service, template_type=EMAIL_TYPE)
     letter_template = sample_template(service=service, template_type=LETTER_TYPE)
-    for year in (2016, 2017):
```

Comment:
Hardcoded, made the two methods that used this incapable of being ran in parallel. Fixed.